terraform/github-wif.tf

```diff
@@ -43,7 +46,7 @@ resource "google_iam_workload_identity_pool_provider" "default" {
   description                        = "${var.name} OIDC identity provider"
 
   attribute_mapping   = local.wif_attribute_mapping
-  attribute_condition = local.wif_attribute_condition
+  attribute_condition = var.github_owner_id != "" && var.github_enterprise_id != "" ? local.wif_attr_org_id + " && " + local.wif_attr_enterprise_id : local.wif_attr_org_id + local.wif_attr_enterprise_id
```

Comment:
I find this a bit confusing, I would add a comment for posterity.

Reply:
I added a comment explaining how that works.